docs/Using Fleet/fleetctl-CLI.md

```diff
@@ -172,75 +110,50 @@ contexts:
     token: your_token_here
 ```
 
-Note the token can also be set with `fleetctl config set --token`, but this may leak the token into a user's shell history.
-
-## Using fleetctl to configure Fleet
-
-A Fleet configuration is defined using one or more declarative "messages" in yaml syntax.
-
-Fleet configuration can be retrieved and applied using the `fleetctl` tool.
-
-### Fleetctl get
-
-The `fleetctl get <fleet-entity-here> > <configuration-file-name-here>.yml` command allows you retrieve the current configuration and create a new file for specified Fleet entity (queries, hosts, etc.)
-
-### Fleetctl apply
-
-The `fleetctl apply -f <configuration-file-name-here>.yml` allows you to apply the current configuration in the specified file. 
-
-When a new configuration is applied, agent options are validated. If any errors are found, you will receive an error message describing the issue and the new configuration will not be applied. You can also verify that your agent options are valid without applying using the `--dry-run` flag. Validation is based on the latest version of osquery. If you don't use the latest version of osquery, you can override validation using the `--force` flag. This will update agent options even if they are invalid.
-
-Check out the [configuration files](https://fleetdm.com/docs/using-fleet/configuration-files) section of the documentation for example yaml files.
+The token can also be set with `fleetctl config set --token`, but this may leak the token into a user's shell history.
 
 ## Using fleetctl with an API-only user
 
-When running automated workflows using the Fleet API, we recommend an API-only user's API key rather than the API key of a regular user. A regular user's API key expires frequently for security purposes, requiring routine updates. Meanwhile, an API-only user's key does not expire.
+When running automated workflows using the Fleet API, we recommend an API-only user's API key rather than the API key of a regular user. Regular user's API key expires frequently for security purposes, requiring routine updates. Meanwhile, an API-only user's key does not expire.   
```

Comment:
```suggestion
When running automated workflows using the Fleet API, we recommend an API-only user's API key rather than the API key of a regular user. A regular user's API key expires frequently for security purposes, requiring routine updates. Meanwhile, an API-only user's key does not expire.   
```